google_bigquery_writer/app.py

```diff
@@ -96,6 +97,8 @@ def get_writer(self):
             project = self.cfg.get_parameters().get('service_account').get('project_id')
         elif self.cfg.get_parameters().get('project'):
             project = self.cfg.get_parameters().get('project')
+        elif self.cfg.config_data.get('image_parameters', {}).get('service_account', {}).get('project_id'):
```

Comment:
This is not needed no? It is already present in credentials_json.

Reply:
Based on the previous code, I suppose that there is a situation where the project is not defined inside the service_account but directly in parameters (for example, when using OAuth, so parameters are not validated). Is it true?

Reply:
Sorry I saw the credentials_json but it belongs to different function (it was wrapped in github). 

The Service account must contain project ID (UI doesn't allow anything else then service account, but leave this logic just in case). 

Anyway this requests purpose is to override the service account for specific stack so the user cannot override it via configuration. => image_parameters must take precendence, when there's service account in image_parameters, everything else is ignored.